selfdrive/car/toyota/values.py

```diff
@@ -1714,4 +1714,4 @@ class CAR:
 NO_DSU_CAR = TSS2_CAR | {CAR.CHR, CAR.CHRH, CAR.CAMRY, CAR.CAMRYH}
 
 # no resume button press required
-NO_STOP_TIMER_CAR = TSS2_CAR | {CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.SIENNA, CAR.LEXUS_ESH}
+NO_STOP_TIMER_CAR = TSS2_CAR | {CAR.PRIUS_V, CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.SIENNA, CAR.LEXUS_ESH}
```

Comment:
are you sure this is right? you don't have to press a button to start going after entering a standstill?

Reply:
I've confirmed that you can check out this TestRoute. 
TestRoute("a2adabc3165216a5|2022-01-25--21-18-16", TOYOTA.PRIUS_V),Route